spec/rails_helper.rb

```diff
@@ -96,10 +96,7 @@
     # Stub required credentials to prevent need for RAILS_MASTER_KEY in test
     allow(EnvironmentCredentials).to receive(:dig).and_call_original
     allow(EnvironmentCredentials).to receive(:dig).with(:db_encryption_key).and_return('any-32-character-string-here!!!!')
-
-    allow_any_instance_of(Intake).to receive(:create_13614c_document)
```

Comment:
Our default setup probably already includes not running jobs in test. We're no longer calling this method explicity in the after update.